src/drawer.js

```diff
@@ -619,16 +623,21 @@ $.Drawer.prototype = {
      */
     setImageSmoothingEnabled: function(imageSmoothingEnabled){
         if ( this.useCanvas ) {
-            var context = this.context;
-            context.mozImageSmoothingEnabled = imageSmoothingEnabled;
-            context.webkitImageSmoothingEnabled = imageSmoothingEnabled;
-            context.msImageSmoothingEnabled = imageSmoothingEnabled;
-            context.imageSmoothingEnabled = imageSmoothingEnabled;
-
+            this._imageSmoothingEnabled = imageSmoothingEnabled;
+            this._setImageSmoothingEnabled();
             this.viewer.forceRedraw();
         }
     },
 
+    // private
+    _setImageSmoothingEnabled: function(){
```

Comment:
I'm thinking `_updateImageSmoothingEnabled` might be a better name for this, since it's just updating the canvas to match the value that's already been set elsewhere. That'll help with any confusion between this function and `setImageSmoothingEnabled` as well. I hope you don't mind the new suggestion (I realize the other name was my suggestion as well).

Reply:
Not at all, I agree and I'm happy to learn...